### PR TITLE
Extend magika wheels test to windows runner

### LINKED ISSUE #805: Extend test "uv add magika.whl" to windows runner
We recently updated our python github workflows to test that the magika package/wheels can be installed with `uv add` (in addition to `pip install magika`). To test this, we use a one-liner that currently works only for ubuntu and macos. We need to extend the one-liner (or add another one) so that it also works for Windows.

These are the workflows to update:
- [ ] https://github.com/google/magika/blob/main/.github/workflows/python-test-published-package.yml
- [ ] https://github.com/google/magika/blob/main/.github/workflows/python-test-published-rc-package.yml
- [ ] https://github.com/google/magika/blob/main/.github/workflows/python-build-package.yml

### PR DESCRIPTION
**Pull request by Mohit-Gaur**
Resolves #805 

Updated the following workflows to install `uv` and verify that `uv add magika.whl` works on windows runner:

* `.github/workflows/python-build-package.yml`
* `.github/workflows/python-test-published-package.yml`
* `.github/workflows/python-test-published-rc-package.yml`

### CODE REVIEW

File: .github/workflows/python-build-package.yml [hunk 1]
```diff
@@ -60,6 +60,19 @@ jobs:
       - if: matrix.platform.runner != 'windows-latest'
         name: Check that `uv add magika.whl` works
         run: mkdir /tmp/test-uv && cp -vR dist/*.whl /tmp/test-uv && cd /tmp/test-uv && uv init && uv add ./$(\ls -1 *.whl | head -n 1)
+      - if: matrix.platform.runner == 'windows-latest'
+        name: Install uv on Windows
```
Comment (reyammer) [inline .github/workflows/python-build-package.yml:64]: do we really need this? Line 37 should already install uv on windows as well? (same for the other two files)

Comment (Mohit-Gaur) [reply .github/workflows/python-build-package.yml:64]: As I understand it, line 37 uses a shell script which will not work on windows. That's why I added a separate step that uses a powershell script to handle uv install.

Comment (reyammer) [reply .github/workflows/python-build-package.yml:64]: From https://github.com/google/magika/actions/runs/12232019796/job/34140885923, it seems that it does run successfully (the command is run for sure as there is no `if` on platform runner OS; question is whether your powershell part can use the just-installed uv). The problematic part was the one-liner starting with `mkdir`, which your PR seems to address. Can you maybe try to remove the "install uv on windows" part and check that it still works?

Comment (Mohit-Gaur) [reply .github/workflows/python-build-package.yml]: Done! I removed that step and it still works fine

**Review by reyammer — APPROVED** [review]

**Pull request merged**